openadapt/strategies/stateful.py

```diff
@@ -96,8 +96,10 @@ def get_next_action_event(
             and not key.endswith("id")
             #and not isinstance(getattr(models.WindowEvent, key), property)
         })
-        reference_window_dict["state"].pop("data")
-        active_window_dict["state"].pop("data")
+        if reference_window_dict and "state" in reference_window_dict:
+            reference_window_dict["state"].pop("data")
+        if active_window_dict and "state" in active_window_dict :
```

Comment:
Please format with `black` 🙏

Reply:
addressed in https://github.com/OpenAdaptAI/OpenAdapt/commit/1027546e62c40ef546e8c8a36050d46d8ff79c97